src/libraries/System.Private.Xml/tests/TrimmingTests/XslCompiledTransformTests.cs

```diff
@@ -0,0 +1,74 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Xml.XPath;
+using System.Xml.Xsl;
+
+class Program
+{
+    static int Main(string[] args)
```

Comment:
Just curious is this only way to test trimming? Cannot have Unit test? Is this test would run with CI build

Reply:
The "TrimmingTests" are run during the CI build with these legs:

![image](https://user-images.githubusercontent.com/8291187/101418447-89f88880-38b3-11eb-8d63-8bb066c362cd.png)

These programs are published, executed, and the infrastructure verifies the application returns exit code `100` - which means "passed".

See https://github.com/dotnet/designs/blob/main/accepted/2020/linking-libraries.md#testing for more information.

Reply:
Cool, thanks!